openquakeplatform/bin/deploy.sh

```diff
@@ -0,0 +1,383 @@
+    
+    sudo sed -i "24 s@^@MEDIA_ROOT = '/var/www/geonode/uploaded'\n@g" /etc/geonode/local_settings.py
+    sudo sed -i "25 s@^@STATIC_ROOT = '/var/www/geonode/static'\n@g" /etc/geonode/local_settings.py
+
+    # pre and post install platform
+    sudo -E ./package/oq_install.sh -s post $HOME/$GIT_REPO/openquakeplatform/common/geonode_install.sh
+    sudo -E ./package/oq_install.sh -s setup_geoserver $HOME/$GIT_REPO/openquakeplatform/common/geonode_install.sh
+    
+    sudo sed -i '1 s@^@WSGIPythonHome '"$HOME"'/env\n@g' /etc/apache2/sites-enabled/geonode.conf
+    sudo invoke-rc.d apache2 restart                      
+}
+
+function apply_data() {
+    cd $HOME
+    geonode syncdb  
+    geonode migrate
+    geonode loaddata $HOME/$GIT_REPO/openquakeplatform/dump/base_topiccategory.json
+    geonode import_vuln_geo_applicability_csv $HOME/$GIT_REPO/openquakeplatform/vulnerability/dev_data/vuln_geo_applicability_data.csv
+    geonode vuln_groups_create
+
+    if [ "$DEVEL_DATA" ]; then
+        geonode add_user $HOME/$GIT_REPO/openquakeplatform/common/gs_data/dump/auth_user.json
+        geonode loaddata $HOME/$GIT_REPO/openquakeplatform/common/gs_data/dump/base_region.json
+        geonode loaddata -v 3 --app vulnerability $HOME/$GIT_REPO/openquakeplatform/common/gs_data/dump/all_vulnerability.json
+        geonode create_gem_user
+    else
+        geonode add_user $HOME/oq-private/old_platform_documents/json/auth_user.json
+        geonode loaddata $HOME/oq-private/old_platform_documents/json/base_region.json
+        geonode loaddata $HOME/$GIT_REPO/openquakeplatform/vulnerability/post_fixtures/initial_data.json
+        geonode loaddata -v 3 --app vulnerability $HOME/oq-private/old_platform_documents/json/all_vulnerability.json
+        geonode create_gem_user
+    fi    
+    pip install simplejson==2.0.9
+    sudo sed -i 's/-Xmx128m/-Xmx4096m/g' /etc/default/tomcat7
+    sudo service tomcat7 restart
+    
+    if [ -z "$DEVEL_DATA" ]; then
+        cp -r $HOME/oq-private/old_platform_documents/thumbs/ /var/www/geonode/uploaded/
+    fi
+
+    cp -r $HOME/$GIT_REPO/openquakeplatform/common/gs_data/documents /var/www/geonode/uploaded/
+
+    cd $HOME/$GIT_REPO
+    if [ "$DEVEL_DATA" ]; then
+
+         ## load data for gec and isc viewer
+         geonode import_isccsv $HOME/$GIT_REPO/openquakeplatform/isc_viewer/dev_data/isc_data.csv  $HOME/$GIT_REPO/openquakeplatform/isc_viewer/dev_data/isc_data_app.csv
+         geonode import_gheccsv $HOME/$GIT_REPO/openquakeplatform/ghec_viewer/dev_data/ghec_data.csv
+
+         # Create programmatically ISC and GHEC json
+         sudo chmod o+w /var/www/geonode/uploaded/thumbs
+         geonode create_iscmap $HOME/$GIT_REPO/openquakeplatform/isc_viewer/dev_data/isc_map_comps.json
+         geonode create_ghecmap $HOME/$GIT_REPO/openquakeplatform/ghec_viewer/dev_data/ghec_map_comps.json
+         sudo chmod o-w /var/www/geonode/uploaded/thumbs
```

Comment:
With the new 664 + g+s permission strategy is **maybe** possible to avoid this ``chmod +w``, ``chmod -w`` procedure.

Reply:
Changed here: https://github.com/gem/oq-platform2/pull/66/commits/90a7f5f06a6a73067d39867625f9c08257119908